main/ertp/api/purse.md

```diff
@@ -1,78 +1,130 @@
 # Purse
-Purses hold an amount of digital assets of the same brand, but unlike payments, they are not meant to be sent to others. To transfer digital assets, a payment should be withdrawn from a purse. A purse's balance can rise and fall, through the action of deposit() and withdraw().
-
-The digital assets in purses and payments can be currency-like and goods-like digital assets, but they can also be other kinds of rights, such as the right to participate in a particular contract.
+`Purses` hold digital assets. They are created to hold assets of a single `brand`,
+and their `brand` cannot be changed. For example, you might create a `purse` to
+hold Quatloos. That `purse` can only hold Quatloos; it can't hold Moola or theater
+tickets or anything else. And you can't change that `purse` so that it now holds
+Moola instead; you need to create a `purse` that holds only Moola.
+
+Digital assets in `purses` and `payments` can be any of:
+- Currency-like, such as our imaginary Quatloos currency.
+- Goods-like digital assets, such as magic weapons for use in a game or theater tickets.
+- Other kinds of rights, such as the right to participate in a particular contract.
+
+While each `purse` can only hold assets of one `brand`, any number of `purses` can be
+created that hold that `brand`. So you could have, say, three Quatloos `purses`, your
+friend Alice could have eight Quatloos `purses`, and so on. 
+
+You change a `purse`'s balance by calling either `deposit()` (to add assets)
+or `withdraw()` (to remove assets) on it. A `purse` can be empty, meaning it
+has a balance value of 0 (which, for a ticket-holding `purse`, means that
+there just aren't any theater tickets in the `purse`).
```

Comment:
I can see that we're trying to accommodate the nonfungible side here too, but can we remove the value of 0? Technically, the value could be []. So although I can see what we're trying to do, I think it is technically incorrect.
```suggestion
or `withdraw()` (to remove assets) on it. A `purse` can be empty. (A purse for a currency
has a balance value of 0, but a ticket-holding purse just doesn't have any tickets).
```

Reply:
How about adding "fungible" to have "A purse for a fungible currency has a balance value of 0, but..." to make the difference clear?

Reply:
sounds good!